Source/DafnyStandardLibraries/src/Std/TargetSpecific/Concurrent-go.dfy

```diff
@@ -6,7 +6,7 @@ module
   class {:extern} MutableMap<K(==), V(==)> ... {
 
     @Axiom
-    constructor {:extern} (ghost inv: (K, V) -> bool)
+    constructor {:extern} (ghost inv: (K, V) -> bool, bytesKeys: bool)
```

Comment:
We define the `bool` param on the `Ctor` - but Go generated from Dafny seems to use `New_X` . I get a compilation error when I try to do this in MPL.